packages/components/src/components/status-badge/status-badge.scss

```diff
@@ -21,10 +20,12 @@
             color: var(--status-warning);
             font-size: var(--text-size-xxxs);
             align-items: center;
+            background-color: var(--status-warning-transparent);
         }
```

Comment:
this will effect all classes. we need to make sure only "pending" class is effected